src/sanitise-old-messages/src/index.ts

```diff
@@ -0,0 +1,35 @@
+import { Client } from "pg"
+import { AuditLogApiClient, AwsAuditLogDynamoGateway, logger } from "shared"
+import { getApiConfig, getDynamoConfig, getPostgresConfig, getSanitiseConfig } from "./config"
+import sanitiseOldMessages from "./sanitiseOldMessages"
+
+export default async (): Promise<void> => {
+  const pgConfig = getPostgresConfig()
+  const dynamoConfig = getDynamoConfig()
+  const apiConfig = getApiConfig()
+  const config = getSanitiseConfig()
+
+  const api = new AuditLogApiClient(apiConfig.API_URL, apiConfig.API_KEY)
+  const dynamo = new AwsAuditLogDynamoGateway(dynamoConfig, dynamoConfig.TABLE_NAME)
+  const db = new Client({
+    host: pgConfig.HOST,
+    port: pgConfig.PORT,
+    user: pgConfig.USERNAME,
+    password: pgConfig.PASSWORD,
+    ssl: pgConfig.SSL
+      ? {
+          rejectUnauthorized: false
+        }
+      : false,
+    database: pgConfig.DATABASE_NAME
+  })
```

Comment:
We can define the above variables (from `pgConfig` to `db`) outside of the function. Lambda will then treat them as global variables and declare and set these at the cold start (first request) and [retain the values between invocations](https://docs.aws.amazon.com/lambda/latest/operatorguide/global-scope.html).

Reply:
Ah cool, didn't know that!

Annoyingly, I don't think we'll be able to do that here. As nice as having our connections global would be, it's quite impractical here. For postgres, we'd need to manage a connection pool rather than just connecting/disconnecting during the lifecycle of the lambda. It also makes setting up the tests a faff, since we'd have to ensure any envrionment variables that our lambda needs are set before `index.ts` is imported.

Think I'm going to leave this for now, unless I'm missing anything glaringly obvious that makes this work nicely?